packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -185,18 +205,27 @@ export class VaButton {
     
     return (
       <Host>
-        <button
-          class={buttonClass}
-          aria-disabled={ariaDisabled}
-          aria-label={label}
-          aria-describedby={ariaDescribedbyIds}
-          type={type}
-          part="button"
-        >
-          {back && !_continue && <va-icon icon="navigate_far_before" />}
-          {buttonText}
-          {_continue && !back && <va-icon icon="navigate_far_next" />}
-        </button>
+          {/* This span must always be present for changes to be announced for the loading prop. It will not show visually or be read without content*/}
```

Comment:
I don't love the idea of this span being rendered for all buttons. I have tested it and it does not announce anything when there is not content, but it would be nice to figure out a way to only render it if the button will be used for loading. Hoping someone has an idea for how to do that.

Reply:
I don't think it's a big deal to have a span always in the markup but was there some limitation with just doing conditional JSX? Something like:

```jsx
{this.loading && (
    <span class="loading-message" role="status">....</span>
)}
```